Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Build the manager binary
+FROM golang:1.19 as builder
```

Comment:
I think we need to use kyma images `eu.gcr.io/kyma-project/external/golang:1.20.1-alpine3.17`
See example from [here](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/Dockerfile)